docs/rpc.md

```diff
@@ -239,6 +240,10 @@ block. It can be removed in future versions, but at the moment you can use it
 to see how much GAS is burned with a particular block (because system fees are
 burned).
 
+#### `getpeerheights` call
+
+This method returns the block height and user agent of all connected peers.
```

Comment:
Please, add a note that block height may be stale a bit and depends on the PingInterval node config.